### ISSUE TITLE
Missing build for Scala 3 + Scala.JS

### ISSUE THREAD

**Issue by japgolly** [scalatest/scalatest#1982]
Hellooooooo! I can see that Scalatest is published for Scala.JS and for Scala 3.0 separately, but is missing a build for Scala 3.0 and Scala.JS together (suffix should be `sjs1_3.0.0-RC1`).

```
> curl -s https://repo1.maven.org/maven2/org/scalatest/ | fgrep scalatest_ | fgrep sjs1_
<a href="scalatest_sjs1_2.11/" title="scalatest_sjs1_2.11/">scalatest_sjs1_2.11/</a>                                             -         -      
<a href="scalatest_sjs1_2.12/" title="scalatest_sjs1_2.12/">scalatest_sjs1_2.12/</a>                                             -         -      
<a href="scalatest_sjs1_2.13/" title="scalatest_sjs1_2.13/">scalatest_sjs1_2.13/</a>                                             -         -      


> curl -s https://repo1.maven.org/maven2/org/scalatest/ | fgrep scalatest_ | fgrep 3.0.0
<a href="scalatest_3.0.0-M1/" title="scalatest_3.0.0-M1/">scalatest_3.0.0-M1/</a>                                              -         -      
<a href="scalatest_3.0.0-M2/" title="scalatest_3.0.0-M2/">scalatest_3.0.0-M2/</a>                                              -         -      
<a href="scalatest_3.0.0-M3/" title="scalatest_3.0.0-M3/">scalatest_3.0.0-M3/</a>                                              -         -      
<a href="scalatest_3.0.0-RC1/" title="scalatest_3.0.0-RC1/">scalatest_3.0.0-RC1/</a>                                             -         -      
```

**Comment by cheeseng** [comment]
@japgolly We were blocked by the following issue in our first attempt: 

https://github.com/scalatest/scalatest/issues/1918#issuecomment-740325472

I just tried it again with the latest 3.0.0-RC1 and the problem seems fixed now, I'll continue with other modules and see if I can build them all and run test.  Hopefully if everything goes fine we may release for Scala 3 + Scala JS.

Cheers.

**Comment by cheeseng** [comment]
@japgolly Fyi PR submitted: 

https://github.com/scalatest/scalatest/pull/1985

**Comment by japgolly** [comment]
Oh wow, that's a very significant amount of work! Thanks so much @cheeseng !

**Comment by cheeseng** [comment]
@japgolly Fyi this is released as part of 3.2.7 already.

Cheers.

**Comment by japgolly** [comment]
Fantastic news, thanks again @cheeseng !!
